app/src/main/res/layout/tool_details_fragment.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <import type="android.text.TextUtils" />
+        <import type="androidx.lifecycle.LiveData" />
+
+        <variable name="fragment" type="org.cru.godtools.ui.tooldetails.ToolDetailsFragment" />
+        <variable name="tool" type="LiveData&lt;org.cru.godtools.model.Tool&gt;" />
+        <variable name="banner" type="LiveData&lt;org.cru.godtools.model.Attachment&gt;" />
+        <variable name="progress" type="org.cru.godtools.download.manager.DownloadProgress" />
+    </data>
+
+    <org.ccci.gto.android.common.widget.HackyNestedScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+        <LinearLayout
```

Comment:
I might convert this to a ConstraintLayout in a subsequent PR. haven't seen if it's worth it yet